index.js

```diff
@@ -226,7 +298,7 @@ function fileStatsEqual(a, b) {
   // Note that stats.mtimeMs is only available as of Node 9
   return (
     a.ino === b.ino &&
-    a.mtime.getTime() === b.mtime.getTime() &&
+    // a.mtime.getTime() === b.mtime.getTime() &&
```

Comment:
is/was this important?

Reply:
@NullVoxPopuli in real life it's really rare case where file is changed without it's changed size